doc/monitoring/when_use_cartridge.rst

```diff
@@ -0,0 +1,221 @@
+.. _when_use_cartridge:
+
+When use Cartridge
```

Comment:
I think something like "Getting started (Cartridge application)" would be a better heading